client/components/breadcrumb/index.tsx

```diff
@@ -28,6 +28,10 @@ const StyledLi = styled.li`
 		--color-link: var( --studio-gray-80 );
 		font-weight: 500;
 	}
+
+	& a {
+		text-decoration: none;
+	}
```

Comment:
@Automattic/red - Since I am making this change outside of the stats app, I need a gut check on this one. This is related to removing underline from breadcrumb https://github.com/Automattic/wp-calypso/pull/102379/files#r2028579346

Reply:
I added this after [I noticed the underline was not present before](https://github.com/Automattic/wp-calypso/issues/89187#issuecomment-2268152069) and probably removed it accidently - as in the above PR is I think removing global rule in favor of the local rule?

Would you happen to know if we have Figma/designs for this layout, and do you know if you know and underline is expected or not here? - cc @kangzj?

Reply:
> I think removing global rule in favor of the local rule?

I think so. We can tackle it under the `.stats-main` layout or somewhere else, as underlines for the breadcrumb might be necessary elsewhere when there are links.

> do you know if you know and underline is expected or not here?

Good question. I recall there being a project for the navigation header refactor, so there might be some design involved. We can still try to make changes only for Stats for now and iterate the component if necessary. 🙂

Reply:
One final question: If you know/have an opinion, do you think we should add a global CSS under `.stats-main a` to disable underlines? For eg, there are underlines for empty states right now which will be removed too if we add a global CSS under stats-main, which I think is expected to be removed - p1HpG7-spP-p2

![Arc -2025-04-12 at 12 38 20@2x](https://github.com/user-attachments/assets/087eac6c-732c-47d8-a084-89512a019732)

Reply:
I went with separate styles in https://github.com/Automattic/wp-calypso/pull/102379/commits/e840a0e857e5007537d1a37fdb7903775cdaca56 following the work being done in https://github.com/Automattic/wp-calypso/pull/101358 to remove global rules in favor of targeting specific selectors https://github.com/Automattic/wp-calypso/pull/101358. Do let me know if you have any thoughts